ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/priorityfee-input.test.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 
+import { GAS_ESTIMATE_TYPES } from '../../../../../shared/constants/gas';
+import { renderWithProvider } from '../../../../../test/lib/render-helpers';
 import mockEstimates from '../../../../../test/data/mock-estimates.json';
 import mockState from '../../../../../test/data/mock-state.json';
-import { renderWithProvider } from '../../../../../test/lib/render-helpers';
-import configureStore from '../../../../store/store';
 import { GasFeeContextProvider } from '../../../../contexts/gasFee';
+import configureStore from '../../../../store/store';
 
-import { GAS_ESTIMATE_TYPES } from '../../../../../shared/constants/gas';
+import { AdvanceGasFeePopoverContextProvider } from '../context';
 import PriprityfeeInput from './priorityfee-input';
```

Comment:
Just curious, why the spelling`PriprityfeeInput` is used here instead of `PriorityfeeInput`?

Reply:
Typo I fixed.